boost-maven/boost-boms/boost-mp22-apis-bom/pom.xml

```diff
@@ -0,0 +1,104 @@
+	<packaging>pom</packaging>
+
+	<description>Booster MP22 APIs Bom</description>
+
+	<properties>
+		<!-- Java EE specs -->
+		<cdi-version>2.0</cdi-version>
+		<jaxrs-version>2.1</jaxrs-version>
+		<jsonp-version>1.1</jsonp-version>
+		<jsonb-version>1.0</jsonb-version>
+		<annotation-version>1.2</annotation-version>
+
+		<!-- MicroProfile specs -->
+		<config-version>1.3</config-version>
+		<ft-version>2.0</ft-version>
+		<health-version>1.0</health-version>
+		<metrics-version>1.1</metrics-version>
+		<jwt-version>1.1</jwt-version>
+		<openapi-version>1.1</openapi-version>
+		<rest-client-version>1.2</rest-client-version>
+		<opentracing-version>1.3</opentracing-version>
+
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>javax.enterprise</groupId>
+				<artifactId>cdi-api</artifactId>
+				<version>${cdi-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.ws.rs</groupId>
+				<artifactId>javax.ws.rs-api</artifactId>
+				<version>${jaxrs-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.json</groupId>
+				<artifactId>javax.json-api</artifactId>
+				<version>${jsonp-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.annotation</groupId>
+				<artifactId>javax.annotation-api</artifactId>
+				<version>${annotation-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.config</groupId>
+				<artifactId>microprofile-config-api</artifactId>
+				<version>${config-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.fault-tolerance</groupId>
+				<artifactId>microprofile-fault-tolerance-api</artifactId>
+				<version>${ft-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.health</groupId>
+				<artifactId>microprofile-health-api</artifactId>
+				<version>${health-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.metrics</groupId>
+				<artifactId>microprofile-metrics-api</artifactId>
+				<version>${metrics-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.jwt</groupId>
+				<artifactId>microprofile-jwt-auth-api</artifactId>
+				<version>${jwt-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.openapi</groupId>
+				<artifactId>microprofile-openapi-api</artifactId>
+				<version>${openapi-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.rest.client</groupId>
+				<artifactId>microprofile-rest-client-api</artifactId>
+				<version>${rest-client-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.opentracing</groupId>
+				<artifactId>microprofile-opentracing-api</artifactId>
+				<version>${opentracing-version}</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
```

Comment:
Noting other comments from https://github.com/dev-tools-for-enterprise-java/boost/issues/339  though we can defer until that issue